afkak/producer.py

```diff
@@ -224,11 +224,11 @@ def stop(self):
         # Do we have to worry about our looping call?
         if self.batch_every_t is not None:
```

Comment:
@rthille I meant that _this_ path is never taken in `test_throughput` because batching isn't enabled. `self._sendLooperD` will always be ``None``.